app/src/main/java/com/hover/stax/views/StaxInfoTextView.kt

```diff
@@ -0,0 +1,46 @@
+package com.hover.stax.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import com.hover.stax.R
+import com.hover.stax.databinding.InfoTextLayoutBinding
+import com.hover.stax.utils.Utils
+
+class StaxInfoTextView(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs) {
```

Comment:
Use plain XML to simplify this. It's a simple component that doesn't introduce any new complex functionality. Consider XML layouts first before creating custom components.